### PR TITLE
Update azure_rm_loganalyticsworkspace related module document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_loganalyticsworkspace related module document.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_loganalyticsworkspace/azure_rm_loganalticsworkspace_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/cloud/azure/azure_rm_loganalyticsworkspace.py [hunk 1]
```diff
@@ -54,21 +54,21 @@
     retention_in_days:
         description:
             - The workspace data retention in days.
-            - -1 means Unlimited retention for the C(unlimited) C(sku).
-            - 730 days is the maximum allowed for all other C(sku)s.
+            - -1 means Unlimited retention for I(sku=unlimited).
+            - 730 days is the maximum allowed for all other skus.
```
Comment (samccann) [inline lib/ansible/modules/cloud/azure/azure_rm_loganalyticsworkspace.py]:
```suggestion
            - 730 days is the maximum allowed for all other SKUs.
```

File: lib/ansible/modules/cloud/azure/azure_rm_loganalyticsworkspace.py [hunk 2]
```diff
@@ -95,15 +96,15 @@
...
     type: str
     returned: success
     example: "per_gb2018"
 retention_in_days:
     description:
         - The workspace data retention in days.
-        - -1 means Unlimited retention for the C(unlimited) C(sku).
-        - 730 days is the maximum allowed for all other C(sku)s.
+        - -1 means Unlimited retention for I(sku=unlimited).
+        - 730 days is the maximum allowed for all other skus.
```
Comment (samccann) [inline lib/ansible/modules/cloud/azure/azure_rm_loganalyticsworkspace.py]:
```suggestion
        - 730 days is the maximum allowed for all other SKUs.
```

File: lib/ansible/modules/cloud/azure/azure_rm_loganalyticsworkspace_facts.py [hunk 3]
```diff
@@ -81,15 +82,15 @@
...
     type: str
     returned: success
     example: "per_gb2018"
 retention_in_days:
     description:
         - The workspace data retention in days.
-        - -1 means Unlimited retention for the C(unlimited) C(sku).
-        - 730 days is the maximum allowed for all other C(sku)s.
+        - -1 means Unlimited retention for I(sku=unlimited).
+        - 730 days is the maximum allowed for all other skus.
```
Comment (samccann) [inline lib/ansible/modules/cloud/azure/azure_rm_loganalyticsworkspace_facts.py]:
```suggestion
        - 730 days is the maximum allowed for all other SKUs.
```

**Review by samccann — COMMENTED** [review]
Thanks for the edits!  I few minor suggestions here.

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_loganalyticsworkspace.py [hunk 4]
```diff
@@ -116,24 +117,24 @@
 management_groups:
     description:
         - List of management groups connected to the workspace.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_loganalyticsworkspace.py]: If this data is returned as a dict, we should change this line as well:

```suggestion
        - Management groups connected to the workspace.
```

File: lib/ansible/modules/cloud/azure/azure_rm_loganalyticsworkspace.py [hunk 5]
```diff
@@ -116,24 +117,24 @@
...
     returned: success
-    example: "{
+    example: {
                 'primarySharedKey': 'BozLY1JnZbxu0jWUQSY8iRPEM8ObmpP8rW+8bUl3+HpDJI+n689SxXgTgU7k1qdxo/WugRLxechxbolAfHM5uA==',
                 'secondarySharedKey': '7tDt5W0JBrCQKtQA3igfFltLSzJeyr9LmuT+B/ibzd8cdC1neZ1ePOQLBx5NUzc0q2VUIK0cLhWNyFvo/hT8Ww=='
-              }"
+              }
 usages:
     description:
         - List of usage metrics for the workspace.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_loganalyticsworkspace.py]: And this one:

```suggestion
        - Usage metrics for the workspace.
```

File: lib/ansible/modules/cloud/azure/azure_rm_loganalyticsworkspace_facts.py [hunk 6]
```diff
@@ -98,28 +99,28 @@
         - Lists all the intelligence packs possible and whether they are enabled or disabled for a given workspace.
     type: list
     returned: success
-    example: ['name': 'CapacityPerformance', 'enabled': true]
+    example: [ {'name': 'CapacityPerformance', 'enabled': true} ]
 management_groups:
     description:
         - List of management groups connected to the workspace.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_loganalyticsworkspace_facts.py]:
```suggestion
        - Management groups connected to the workspace.
```

File: lib/ansible/modules/cloud/azure/azure_rm_loganalyticsworkspace_facts.py [hunk 7]
```diff
@@ -98,28 +99,28 @@
...
     returned: success
-    example: "{
+    example: {
                 'primarySharedKey': 'BozLY1JnZbxu0jWUQSY8iRPEM8ObmpP8rW+8bUl3+HpDJI+n689SxXgTgU7k1qdxo/WugRLxechxbolAfHM5uA==',
                 'secondarySharedKey': '7tDt5W0JBrCQKtQA3igfFltLSzJeyr9LmuT+B/ibzd8cdC1neZ1ePOQLBx5NUzc0q2VUIK0cLhWNyFvo/hT8Ww=='
-              }"
+              }
 usages:
     description:
         - List of usage metrics for the workspace.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_loganalyticsworkspace_facts.py]:
```suggestion
        - Usage metrics for the workspace.
```

**Review by acozine — COMMENTED** [review]
Found a couple more small changes that will help clarify the docs for these modules.

**Comment by acozine** [comment]
Thanks @Fred-sun @samccann!

**Pull request merged**